src/Bridge.jl

```diff
@@ -556,6 +573,10 @@ function _unmissing!(x::Ref, v::Val{T}) where {T}
 end
 
 function _unmissing!(x::Vector, v::Val{T}) where {T}
+    # a single NaN value is an empty TSeries
+    if length(x) == 1 && _ismissing(x[1], v)
+        return typeof(x)()
```

Comment:
I suggest `return empty!(x)` -- mutate the existing `x` according to the trailing ! convention. 

Although, looking at how `_unmissing!()` is actually used, your `return typeof(x)()` will also work correctly.